package.json

```diff
@@ -1463,6 +1464,13 @@
                     "group": "1_view",
                     "when": "debugProtocolVariableMenuContext == 'viewableInDataViewer'"
                 }
+            ],
+            "editor/title/run": [
+                {
+                    "command": "jupyter.runFileInteractive",
+                    "group": "jupyter",
+                    "when": "resourceLangId == python && !isInDiffEditor && !notebookEditorFocused && isWorkspaceTrusted"
```

Comment:
So this can work for any python file, even if it doesn't have `#%%`?

Reply:
And is there a debug option?

Reply:
correct, cells are not required.
No current debug option - debugFileInteractive has very low usage relative to runFile or debugCell, though perhaps there is an argument for discoverability to be made.